consumer.go

```diff
@@ -659,7 +659,7 @@ func (c *Consumer) fetchOffsets(subs map[string][]int32) (map[string]map[int32]o
 	}
 
 	// Wait for other cluster consumers to process, release and commit
-	time.Sleep(c.client.config.Consumer.MaxProcessingTime * 2)
+	time.Sleep(*(c.client.config.Group.Offsets.Synchronization.DwellTime) * 2)
```

Comment:
Since this is potentially a substantial delay, I would consider replacing `time.Sleep` with something like this instead:

```
select { 
case <-c.dying:
  return nil, errConsumerClosing
case <-time.After(c.client.config.Group.Offsets.Synchronization.DwellTime*2):
}
```